src/jsx/pages/Profiles/profileData.js

```diff
@@ -0,0 +1,65 @@
+
+
+const profileTable = {
+    data: [
+        [
+            "ScubaTuba32",
+            "brantfisch@scub.atu",
+            "Brant Fischer",
+            "Intermediate",
+            "Parent",
+         ],
+         [
+          "puddinpop12",
+          "quinn_harley@yahoo.com",
+          "Harley Quinn",
+          "Beginner",
+          "Student",
+         ],
+         [
+          "GoldStandard783",
+          "mansa@musa.net",
+          "Mansa Musa",
+          "Intermediate",
+          "Teacher",
+         ],
+       [
+          "DreamTeamCaptain",
+          "mlk@cogic.com",
+          "Martin King",
+          "Intermediate",
+          "Parent",
+       ],
+       [
+        "Marleezy419",
+        "marleezy@gmail.com",
+        "Damien Marley",
+        "Beginner",
+        "Student",
+       ],
+       [
+        "FScottLaGerald",
+        "fsg@hotmail.com",
+        "Scott La Rock",
+        "Intermediate",
+        "Teacher",
+       ],
+       [
+          "FightingTCell68",
+          "tcell@hotmail.com",
+          "Joe Frazier",
+          "Expert",
+          "Student",
+       ],
+    ],
+    columns: [
+       "Username",
+       "Email",
+       "Name",
+       "Certificate Level",
+       "Role",
+// could we change the order of these potentially. also, would certificate level show for every role?
```

Comment:
Could we change the order of these columns? I just put them in with the order that was on the ticket. 
Also, would certificate level show for every role?